src/SourceGenerators/Uno.UI.SourceGenerators/XamlGenerator/XamlFileGenerator.Reflection.cs

```diff
@@ -711,12 +711,6 @@ private INamedTypeSymbol GetType(XamlType type)
 		private static string GetTrimmedNamespace(string nsNamespace)
 		{
 			var nsName = nsNamespace.TrimStart("using:");
-
-			if (nsName.StartsWith("clr-namespace:", StringComparison.Ordinal))
-			{
-				nsName = nsName.Split(';')[0].TrimStart("clr-namespace:");
-			}
```

Comment:
This may be breaking, and I think that Windows _may_ be supporting it.

Reply:
I wasn't able to get it to work on WinUI, but reverting to be on the safe side.

Reply:
We can add it as a breaking change for the next release though.

Reply:
Added to #8339